### PR TITLE
Feature/clean test

### PR DESCRIPTION
**Pull request by dmiracle**
remove vestigial notebooks and deprecated hardware

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM python
+COPY . /app
+RUN python3 -m pip install --user pipx && python3 -m pipx ensurepath
```
Comment (jjalfaro9) [inline Dockerfile:3]: Not sure, so better for me to ask. should this be `python` instead of `python3`?

**Review by jjalfaro9 — COMMENTED** [review]
the code cleanup & readme look good. Just have this one minimal question about your docker file. have you built the image locally?

File: Dockerfile (hunk 1 above)
Comment (dmiracle) [reply Dockerfile:3]: both are interchangeable is my understanding in any python3 image, but we should probably be consistent, I will change to python

**Comment by dmiracle** [comment]
> the code cleanup & readme look good. Just have this one minimal question about your docker file. have you built the image locally?

yes -- removing all the testing stuff and will do that in a new pr

**Pull request merged**